content/klc/S2.3.adoc

```diff
@@ -1,21 +1,18 @@
 +++
-brief = "Where atomic parts are available in multiple footprint options, a separate symbol must be drawn for each footprint"
+brief = "Where parts are available in multiple footprint options, a separate symbol must be drawn for each footprint"
 +++
 
-Many electronic components are provided in multiple footprint (package) options. These _may or may not_ be pin compatible.
+Many electronic components are provided in multiple packaging variants. These _may or may not_ be pin compatible.
+
+KiCad is not able to assign different footprints for aliases. Fully specified symbols therefore require a separate symbol for every package variant. Similarly kicad is not able to define different 3d models other than having a separate footprint. This means a part needs a separate footprint even if the land pattern is the same if the 3d model has functional differences (example different height).
```

Comment:
Same comment as above. Add a comma after 'Similarly'. There are missing capital letters in 'KiCad' and '3D'. I might even re-write the last two sentences to 'Similarly, KiCad a footprint has a 1:1 relationship with its 3D model. This means a multiple footprints are needed where the 3D model has functional differences even if the footprints are identical (for example when 3D model height differs).'.

Reply:
Reworded inspired by your suggestion but removed some things that looked wrong to me.